sources/packages/api/src/services/application/application.service.ts

```diff
@@ -289,6 +289,18 @@ export class ApplicationService extends RecordDataModelService<Application> {
     return application;
   }
 
+  async getApplicationById(applicationId: number): Promise<Application> {
```

Comment:
Just a suggestion. We get application by id and program year but the method name describes only id. Service method name could be changed.

Reply:
IMO this method could keep the generic name as it is. If we need to get more information we can add more items to the select. I would not recommend creating a specific getApplicationX to every single select that we need unless there is a huge performance impact for a particular feature.

Reply:
Agreed @andrewsignori-aot . Actually we are not getting the application with respect to the program year. The where clause contains if the program year is active. We can have a discussion if you want.

Reply:
Not required @guru-aot @andrewsignori-aot. I am good.